packages/remark-lint-emphasis-marker/index.js

```diff
@@ -40,32 +40,32 @@
  * @copyright 2015 Titus Wormer
  * @license MIT
  * @example
- *   {"setting": "*", "name": "ok.md"}
+ *   {"settings": {"emphasis": "*"}, "name": "ok.md"}
```

Comment:
this appears to suggest a breaking change?
That all configuration would need to be updated to this format?
Is that accurate?